lib/parse.js

```diff
@@ -2541,6 +2562,12 @@ function parse($TEXT, options) {
           case "atom":
             next();
             return tmp.value;
+          case "expand":
+            next();
+            return new AST_Expansion({
+                start: prev(),
+                expression: expression(false),
+            });
```

Comment:
fixed in https://github.com/mishoo/UglifyJS2/pull/2265/commits/a7e6eb43a9f9c6897e8b14a8ac536b80e795de04